src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -88,6 +90,15 @@ public ReadOnlyAddressBook getAddressBook() {
         return model.getMatchList();
     }
 
+    @Override
+    public ObservableList<Person> getReminderPersonList() {
+        ObservableList<Person> reminderList = FXCollections.observableArrayList();
+        ReminderPersons reminderPersons = ReminderPersons.getInstance();
+        @SuppressWarnings("unchecked") HashSet<Person> reminderPersonsList = (HashSet<Person>) reminderPersons.clone();
```

Comment:
Perhaps add a comment on why you need to `suppresswarnings` here

Reply:
thanks for pointing that out Howard! 

as Bryan had suggested an alternative to getting a copy of the `HashSet`, i won't need the `SupressWarnings()` anymore, will remove it in the next commit